src/pages/Signup.jsx

```diff
@@ -1,7 +1,204 @@
+      !validationMessage.firstName &&
+      !validationMessage.lastName &&
+      !validationMessage.email &&
+      !validationMessage.password &&
+      !validationMessage.confirmPassword
+    ) {
+      console.log("status ke pehle");
+      const status = await registerUser(signUpForm);
+      console.log(status);
+      if (status === 201) navigate("/login");
+      else {
+        console.log("already registered");
+      }
+
+      console.log("shashi");
+    } else {
+    }
+  };
+
+  return (
+    <main className="signup-container">
+      <section className="signup-box">
+        <h1 className="signup-header">Sign- Up</h1>
+        <div className="lab-input">
+          <label>FirstName*</label>
+          <input
+            type="text"
+            placeholder="shashi"
+            value={signUpForm.firstName}
+            onChange={(e) =>
+              updateSignUpForm({ ...signUpForm, firstName: e.target.value })
+            }
+          />
+          <p className="form_error">{validationMessage.firstName}</p>
+        </div>
+        <div className="lab-input">
+          <label>LastName*</label>
+          <input
+            type="text"
+            placeholder="Maurya"
+            value={signUpForm.lastName}
+            onChange={(e) =>
+              updateSignUpForm({ ...signUpForm, lastName: e.target.value })
+            }
+          />
+          <p className="form_error">{validationMessage.lastName}</p>
+        </div>
+        <div className="lab-input">
+          <label>Email*</label>
+          <input
+            type="Email"
+            placeholder="shashimourya1@gmail.com"
+            value={signUpForm.email}
+            onChange={(e) =>
+              updateSignUpForm({ ...signUpForm, email: e.target.value })
+            }
+          />
+          <p className="form_error">{validationMessage.email}</p>
+        </div>
+        <div className="lab-input">
+          <label> Password*</label>
+          <input
+            type="password"
+            placeholder="Do not share your password                                                    👀 "
+            value={signUpForm.password}
+            onChange={(e) =>
+              updateSignUpForm({ ...signUpForm, password: e.target.value })
+            }
+          />
+          <p className="form_error">{validationMessage.password}</p>
+        </div>
+        <div className="lab-input">
+          <label>Confirm Password*</label>
+          <input
+            type="password"
+            placeholder="Do not share your password                                                    👀"
+            onChange={(e) =>
+              updateSignUpForm({
+                ...signUpForm,
+                confirmPassword: e.target.value,
+              })
+            }
+            value={signUpForm.confirmPassword}
+          />
+          <p>{validationMessage.confirmPassword}</p>
+        </div>
+
+        <div className="checkbox" style={{ width: "90%" }}>
```

Comment:
avoid inline styles

Reply:
it was particualr to that element so creating a class was more effort.i will keep this suggestion for my future pr